tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/ModuleUtil.java

```diff
@@ -59,6 +80,9 @@ public AddModuleHandler resource(String jarName, Class... classes) {
             return this;
         }
 
+        /**
+         * Execute the oppertaion on {@param client}. ENV property {@code JBOSS_HOME} must be set.
```

Comment:
> ENV property {@code JBOSS_HOME} must be set.

What requires this? How is it set in this TS, we use `jboss.home` system property.

Reply:
Module add operation needs it. `JBOSS_HOME` is set in root pom.xml to `jboss.home` value.

Reply:
@istraka thanks